docs/source/tracking.rst

```diff
@@ -180,6 +178,77 @@ and the `LocalArtifactRepository <https://github.com/mlflow/mlflow/blob/master/m
 `S3ArtifactRepository <https://github.com/mlflow/mlflow/blob/master/mlflow/store/artifact/s3_artifact_repo.py#L14>`_ are
 concrete implementations of the abstract class `ArtifactRepository <https://github.com/mlflow/mlflow/blob/master/mlflow/store/artifact/artifact_repo.py#L13>`_.
 
+Scenario 5: MLflow Tracking Server enabled as proxy for artifact storage access
+-------------------------------------------------------------------------------
+
+MLflow's Tracking Server supports utilizing the host as a proxy server for operations involving artifacts.
+Once configured with the appropriate access requirements, an administrator can start the tracking server to enable assumed-role operations involving the saving, loading, or listing of
+model artifacts, images, documents, and files. This eliminates the need to allow end users to have direct path access to a remote file store for artifact handling and eliminates the
+need for an end-user to provide access credentials to interact with an underlying file store.
```

Comment:
Nit: "File store" is a bit of an overloaded term in MLflow, generally referring to the `FileStore` implementation of the backend metadata store API. Can we use "object store" instead?

Reply:
fixed and added references to object stores (just in case)